apps/election-manager/test/renderInAppContext.tsx

```diff
@@ -24,22 +21,14 @@ import CastVoteRecordFiles, {
 import { UsbDriveStatus } from '../src/lib/usbstick'
 import { getEmptyFullElectionTally } from '../src/lib/votecounting'
 
-const eitherNeitherElectionData = fs.readFileSync(
-  join(
-    __dirname,
-    'fixtures/eitherneither-election/eitherneither-election.json'
-  ),
-  'utf-8'
-)
-const eitherNeitherElectionHash = sha256(eitherNeitherElectionData)
-const eitherNeitherElection = parseElection(
-  JSON.parse(eitherNeitherElectionData)
-)
+const eitherNeitherElection =
+  electionWithMsEitherNeitherInternal.electionDefinition.election
 
-export const defaultElectionDefinition = {
+export const eitherNeitherElectionDefinition = {
   election: eitherNeitherElection,
-  electionData: eitherNeitherElectionData,
-  electionHash: eitherNeitherElectionHash,
+  electionData: JSON.stringify(eitherNeitherElection),
```

Comment:
We shouldn't be generating this separately from the hash or they'll be out of sync.